3D-microservices-observability/springboot-jersey-app-2/README.md

```diff
@@ -0,0 +1,78 @@
+# Wavefront Distributed Tracing Sample Apps - Springboot Jersey App 2
+
+This is a sample java application using Springboot beachshirts (#[beachops](https://medium.com/@matthewzeier/thoughts-from-an-operations-wrangler-how-we-use-alerts-to-monitor-wavefront-71329c5e57a8)) 
+which makes shirts for the beach. The application is based on spring boot web using rest templates and rest annotations.
+- JDK 1.8 and up
+- Spring Boot 2.3.0.RC1
+
+## Running Application locally 
+
+- `git clone` this repo and navigate to this dir:
+
+- ```bash
+  git clone https://github.com/howardyoo/distributed-tracing-sample-apps/
+  cd ./distributed-tracing-sample-apps/3D-microservices-observability/springboot-jersey-app-2
+  ```
+- Run `mvn clean compile` from the root directory of the project.
```

Comment:
Running the command was successful, but gave this warning at the beginning:

```
[INFO] Scanning for projects...
[WARNING]
[WARNING] Some problems were encountered while building the effective model for com.wavefront:common:jar:0.0.1-SNAPSHOT
[WARNING] 'dependencies.dependency.version' for org.springframework.cloud:spring-cloud-starter-eureka-server:jar is either LATEST or RELEASE (both of them are being deprecated) @ com.wavefront:springboot-jersey-app-2:0.0.1-SNAPSHOT, /Users/oppegard/workspace/wf/distributed-tracing-sample-apps/3D-microservices-observability/springboot-jersey-app-2/pom.xml, line 88, column 13
[WARNING]
[WARNING] Some problems were encountered while building the effective model for com.wavefront:springboot-jersey-app-2:pom:0.0.1-SNAPSHOT
[WARNING] 'dependencies.dependency.version' for org.springframework.cloud:spring-cloud-starter-eureka-server:jar is either LATEST or RELEASE (both of them are being deprecated) @ line 88, column 13
[WARNING]
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING]
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
[WARNING]
```
Can you take a look and see if this is fixable?